server/service/src/sync/translations/invoice.rs

```diff
@@ -433,12 +453,6 @@ impl SyncTranslation for InvoiceTranslation {
 
         let json_record = serde_json::to_value(legacy_row)?;
 
-        // log::info!(
```

Comment:
Not sure why this was here, but is commented out so thought it was okay to remove